pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -98,13 +98,25 @@ func validateServiceBrokerSpec(spec *sc.ServiceBrokerSpec, fldPath *field.Path)
 				field.Required(fldPath.Child("authInfo"), "auth config is required"),
 			)
 		}
-
 	}
 
 	if spec.InsecureSkipTLSVerify && len(spec.CABundle) > 0 {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("caBundle"), spec.CABundle, "caBundle cannot be used when insecureSkipTLSVerify is true"))
 	}
 
+	if "" == spec.RelistBehavior {
+		allErrs = append(allErrs,
+			field.Required(fldPath.Child("relistDuration"),
+				"brokers must have a relistDuration"))
+	}
+
+	if spec.RelistBehavior == sc.ServiceBrokerRelistBehaviorDuration && spec.RelistDuration == nil {
```

Comment:
We need to validate that the RelistDuration is above a reasonable minimum, at least above 0.